docker/images/snapshot/Dockerfile

```diff
@@ -1,26 +1,12 @@
 # skycoin build binaries
```

Comment:
Is it possible to have a single Dockerfile to build both them images ?

Reply:
It could be possible, but the user would have to checkout to the intended tag manually. The only difference between the two dockerfiles is where do they get the code from: snapshot uses the working copy and mainnet downloads the source from github. You just tell me what to do.

Reply:
The source packages downloaded from github.com are built from tags . Repository check out could work for both cases , for instance , by having release tag name passed in as build / run argument , you'll get same files as if you had downloaded the source package .

Reply:
`https://github.com/skycoin/skycoin/archive/$branch_or_tag.zip`
`https://github.com/skycoin/skycoin/archive/$branch_or_tag.tar.gz`
`git clone https://github.com/skycoin/skycoin.git; git checkout $branch_or_tag`

These all work to get the source for `master`, `develop`, `v0.22.0` etc.  Don't need multiple methods of acquiring the source.  Downloading the zip or tar.gz should be faster and take less space.

Reply:
👍

Reply:
OK, I'll unify them. As the repository will be on the build computer already, I'll just copy it to the compile container, checkout to the specified branch/tag/revision, if passed using `--build-args`, and build it. If nothing is specified, it will build the working copy.

Reply:
We can't assume the repo is on the build computer before calling docker, if they are fetching the image from docker hub

Reply:
If you are fetching the image from docker hub, the image is already built. If you are going to build the image, you need the Dockerfile and it is in the repo. The only case I can see this failing, is somebody downloading just the Dockerfile and trying to build it alone. It seems to me a very edge case. Anyway just tell me what to do and I'll do it.